css/style.css

```diff
@@ -9,400 +14,609 @@ body {
+    height: 60%;
+     }
+}
+ 
+@media (max-width: 1500px) {
+  .new-сollection__main-card {
+    width: 38%;
+    left: 55%;
+    margin-top: 1%;
+     }
+}
+     
+@media (max-width: 1500px) {
+  .girl {
+    width: 50%;
+    right: 60%;
+     }
+}
 
-@media (max-width: 630px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: -50px 92px; 
+@media (max-width: 1500px) {
+  .new-сollection__social {
+    width: 45.5%;
+   }
+}
+
+@media (max-width: 1300px) {
+  .new-сollection__social {
+    width: 47%;
+   }
+}
+
+@media (min-width: 1270px) {
+  .body {
+    margin: 0 auto;
+   }
+}
+
+@media (max-width: 1200px) { 
+  .new-сollection__social {
+    width: 45%;
     }
- }
+}
 
- @media (max-width: 630px) { 
-    .title {
-        top: 83%;
-        width: 15%;
-        font-size: 40px;
-        line-height: 42px;
-        left: 20%;
+@media (max-width: 1200px) { 
+  .girl {
+    width: 45%;
+    right: 60%;
     }
+}
+ 
+@media (max-width: 1200px) { 
+  .new-сollection__main-button {
+    width: 35%;
+    font-size: 8px;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+  .new-сollection__main-link {
+    width: 50%;
+    margin-left: 3%;
+    font-size: 10px;
+    margin-right: -15%;
+     }
+}
+ 
+@media (max-width: 1200px) {
+  .new-сollection-arrow {
+    width: 10%;
      } 
+}
 
-@media (max-width: 630px) {      
-     .galka  {
-        position: absolute;
-        left: 45%;
-        top: 95%;
+@media (max-width: 1200px) {
+  .new-сollection__social {
+    width: 47%;
+     }
+}
+
+@media (max-width: 961px) { 
+  .new-сollection__main-card {
+    margin-top: 5%;
     }
-} 
+}
 
-@media (max-width: 499px) { 
-    .title {
-        top: 65%;
-        font-size: 25px;
-        line-height: 22px;
+@media (max-width: 961px) { 
+  .new-сollection__main-link {
+    margin-right: -5%;
     }
-     } 
+}
 
-@media (max-width: 499px) {      
-     .galka  {
-        left: 40%;
-        top: 75%;
+@media (max-width: 961px) { 
+  .new-collection__background {
+    max-width: 45%;
+    min-height: 900px;
     }
-} 
+}
 
-@media (max-width: 499px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-position: 20% 36%; 
-        margin-left: 10%;
+@media (max-width: 961px) {
+  .logo {
+    margin-right: 20%;
     }
- }
+}
+
+@media (max-width: 961px) {
+  .new-сollection-main-link {
+    margin-left: 5%;
+   } 
+}
+
+@media (min-width: 800px) { 
+  .logo-mobil,
+  .galka,
+  .new-collection__background_girl {
+    display: none;
+    } 
+} 
+ 
+@media (max-width: 799px) { 
+  .logo,
+  .lang,
+  .check,
+  .new-сollection__social,
+  .text,
+  .new-сollection__main-card,
+  .new-collection__background {
+    display: none;
+    } 
+} 
+ 
+@media (max-width: 800px) { 
+  .logo-mobil {
+    position: absolute;
+    width: 98px;
+    height: 37px;
+    padding: 5% 0 0 7%;
+    }
+}  
+ 
+@media (max-width: 799px) { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)